src/services/tasks/schedules/types.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  AffectedRecords,
+  ObjectId,
+  OptionsBase,
+  OptionsWithRql,
+  PagedResultWithPager,
+} from '../../types';
+import { FindAllIterator } from '../../helpers';
+
+export interface SchedulesService {
+  /**
+   * ## Create a new Schedule
+   *
+   * **Global Permissions:**
+   * - `CREATE_TASK_SCHEDULE` - Allows a user to create schedules
+   *
+   * @param schedule - The data used to create the schedule
+   * @param options - Additional options for the request
+   */
+  create<T>(
+    schedule: ScheduleCreation<T>,
+    options?: OptionsBase
+  ): Promise<ScheduleCreation<T>>;
```

Comment:
This response should actually return of type `Schedule` sadly the service is incorrect and returns `ScheduleCreation` thus, we do the same in the SDK.

A ticket has been made to address the service: https://extrahorizon.atlassian.net/browse/SVCS-1834